pyccel/codegen/printing/ccode.py

```diff
@@ -104,6 +109,82 @@ def _traverse_matrix_indices(self, mat):
     def _print_Module(self, expr):
         return '\n\n'.join(self._print(i) for i in expr.body)
 
+    def _print_While(self,expr):
+        code = "while (%s)\n{" % self._print(expr.test)
+        code = code + "\n %s" % self._print(expr.body) + "\n}"
+        return (code)
+
+    def _print_If(self, expr):
+        lines = []
+        for i, (c, e) in enumerate(expr.args):
+            var = self._print(e)
+            #change this line (removes an extra empty else{})
```

Comment:
Is this a `TODO`? If not, then what is being changed in the line?

Reply:
Yes this was a TODO, I just forgot it there, I will remove it